marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/CreateAndUpdateModelTest.java

```diff
@@ -207,6 +221,101 @@ private void updateModelEntityTypes() {
+            "          \"" + ENTITY_PROPERTY_1 + "\"\n" +
+            "        ],\n" +
+            "        \"elementRangeIndex\": [\n" +
+            "          \"" + ENTITY_PROPERTY_1 + "\"\n" +
+            "        ],\n" +
+            "        \"properties\": {\n" +
+            "          \"" + ENTITY_PROPERTY_1 + "\": {\n" +
+            "            \"datatype\": \"dateTime\",\n" +
+            "            \"facetable\": true,\n" +
+            "            \"collation\": \"http://marklogic.com/collation/codepoint\"\n" +
+            "          }\n" +
+            "        }\n" +
+            "      }\n" +
+            "    }\n" +
+            "  }\n" +
+            "]";
+
+        controller.updateModelEntityTypes(readJsonArray(entityTypes));
+    }
+
+    private void updateDataType() {
+        Assumptions.assumeTrue(isVersionCompatibleWith520Roles());
+
+        String entityTypes = "[\n" +
+            "  {\n" +
+            "    \"entityName\": \"" + MODEL_NAME + "\",\n" +
+            "    \"modelDefinition\": {\n" +
+            "      \"Customer\": {\n" +
+            "        \"required\": [],\n" +
+            "        \"pii\": [\n" +
+            "          \"" + ENTITY_PROPERTY_1 + "\"\n" +
+            "        ],\n" +
+            "        \"elementRangeIndex\": [\n" +
+            "          \"" + ENTITY_PROPERTY_1 + "\"\n" +
+            "        ],\n" +
+            "        \"properties\": {\n" +
+            "          \"" + ENTITY_PROPERTY_1 + "\": {\n" +
+            "            \"datatype\": \"date\",\n" +
+            "            \"facetable\": true,\n" +
+            "            \"collation\": \"http://marklogic.com/collation/codepoint\"\n" +
+            "          }\n" +
+            "        }\n" +
+            "      }\n" +
+            "    }\n" +
+            "  }\n" +
+            "]";
+
+        controller.updateModelEntityTypes(readJsonArray(entityTypes));
+    }
+
+    private void addProperty() {
+        Assumptions.assumeTrue(isVersionCompatibleWith520Roles());
+
+        String entityTypes = "[\n" +
+            "  {\n" +
+            "    \"entityName\": \"" + MODEL_NAME + "\",\n" +
+            "    \"modelDefinition\": {\n" +
+            "      \"Customer\": {\n" +
+            "        \"required\": [],\n" +
+            "        \"pii\": [\n" +
+            "          \"" + ENTITY_PROPERTY_1 + "\"\n" +
+            "        ],\n" +
+            "        \"elementRangeIndex\": [\n" +
+            "          \"" + ENTITY_PROPERTY_1 + "\"\n" +
+            "        ],\n" +
+            "        \"properties\": {\n" +
+            "          \"" + ENTITY_PROPERTY_1 + "\": {\n" +
+            "            \"datatype\": \"date\",\n" +
+            "            \"facetable\": true,\n" +
+            "            \"collation\": \"http://marklogic.com/collation/codepoint\"\n" +
+            "          },\n" +
+            "          \"" + ENTITY_PROPERTY_2 + "\": {\n" +
```

Comment:
Same thing here - minimize the payload, and remove facetable/collation for the new property, as whether or not the new property is facetable or not is irrelevant. We want to make it easy as possible to see that the only things that are changing are 1) the datatype in the first update, and then 2) the addition of a new property in the second update.